### PR TITLE
Hide status bar on visible->hidden transition

### PR DESCRIPTION
**Pull request by t3chguy**
`RoomStatusBar` takes an `onHidden` prop yet never uses it...until now

Signed-off-by: Michael Telatynski <7t3chguy@gmail.com>

### CODE REVIEW

**Comment by lukebarnard1** [comment]
This is (weirdly) why the status bar doesn't disappear once it's been expanded. This is actually the desired behaviour. However, it would be nice to clean this mess up a bit.

@t3chguy perhaps you could also comment out the code in `RoomView.onStatusBarHidden` to maintain current functionality and leave a comment explaining that we don't want the status bar to collapse because it turns out that it's really annoying to have it expanding and collapsing all the time.

**Comment by t3chguy** [comment]
@lukebarnard1 so to confirm, this PR is wrong and instead it should just get rid of `onHidden` and `onStatusBarHidden` and put comments in their place?

**Comment by lukebarnard1** [comment]
@t3chguy due to the temporary nature of the functionality (we want something much better in future) I would just comment out the code in `RoomView.onStatusBarHidden` and leave a comment there saying `TODO: Find a less annoying way of hiding the status bar`. 

Your existing commit is fine.

File: src/components/structures/RoomView.js [hunk 1]
```diff
@@ -1347,10 +1347,12 @@ module.exports = React.createClass({
     },
 
     onStatusBarHidden: function() {
-        if (this.unmounted) return;
+        // This is currently nor desired as it is annoying if it keeps expanding and collapsing
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply src/components/structures/RoomView.js]: heh, JetBrains has spellcheck, but if only there was a Grammarly integ to find typos which are still words

**Pull request merged**